db/migrate/20190305150816_drop_elements_from_vm_attrs_hash.rb

```diff
@@ -0,0 +1,68 @@
+class DropElementsFromVmAttrsHash < ActiveRecord::Migration[5.1]
+  def up
+    say "Formatting nics attributes, total #{FakeNic.unscoped.count}"
+    FakeNic.unscoped.each do |compute_attribute|
```

Comment:
you can use `find_each` instead of `each` so you don't load all of the table to memory but just chunks of 1000 records (some tables can become quite large).
You can also prefilter this to only affected resources using 
```ruby
 FakeNic.unscoped.where("attrs like ? or compute_attributes like ?", "%elements:%", "%elements:%").find_each...
```